lfucg-exactions/client/js/components/SubdivisionForm.js

```diff
@@ -105,9 +102,7 @@ SubdivisionForm.propTypes = {
 
 function mapStateToProps(state) {
     return {
-        currentUser: state.currentUser,
         activeForm: state.activeForm,
-        subdivisions: state.subdivisions,
```

Comment:
Do we not need subdivisions in the state for updating forms?

Reply:
Am I understanding this wrong? I though this takes what is in the state and passes it to the props of the of the component we are in. We weren't not using either of these so I went forward with cleaning up the code. Didn't see a change in any functionality but I could very well be understanding what is happening here.

Reply:
Okay.